src/Generator.php

```diff
@@ -69,6 +70,8 @@ public function generateClass(string $className, string $templateName, array $va
 
         $this->addOperation($targetPath, $templateName, $variables);
 
+        $this->generatedPhpFiles[] = $targetPath;
+
```

Comment:
We put this here, but not in, for example, `generateFile()`? Perhaps this should be done in `writeChanges()`? Each time we process a pending operation, we add it to the array. We could filter it later for files that end in `.php`.

Reply:
Good call and done - this will make it easier to implement the twig linter in a future pr.